core/test/scripts/Voting.js

```diff
@@ -147,4 +147,47 @@ contract("scripts/Voting.js", function(accounts) {
     assert.equal(await voting.getPrice(identifier1, time2), hardcodedPrice);
     assert.equal(await voting.getPrice(identifier2, time2), hardcodedPrice);
   });
+
+  it.only("CryptoCompare price", async function() {
+    const identifier = web3.utils.utf8ToHex("BTCUSD");
+    const time = "1560762000";
+
+    // Request an Oracle price.
+    await voting.addSupportedIdentifier(identifier);
+    await voting.requestPrice(identifier, time);
+
+    // Move to the round in which voters will vote on the requested price.
+    await moveToNextRound(voting);
+
+    // Sanity check.
+    assert.isFalse(await voting.hasPrice(identifier, time));
+
+    const emailSender = new MockEmailSender();
+    let votingSystem = new VotingScript.VotingSystem(voting, encryptedSender, voter, emailSender);
+
+    assert.equal(emailSender.emailsSent, 0);
```

Comment:
nit: I think we can remove all of these email checks since this test isn't really testing that.